react-native-libraries.json

```
@@ -11747,5 +11747,18 @@
     "android": true,
     "unmaintained": true,
     "newArchitecture": false
+  },
+  {
+    "githubUrl": "https://github.com/aptabase/aptabase-react-native",
+    "npmPkg": "https://www.npmjs.com/package/@aptabase/react-native",
```

Hello @ivnbogdan, thanks for adding new entry to the directory! 👍 

The change below should fix the failing CI:

```suggestion
    "npmPkg": "@aptabase/react-native",
```